golang/outbound.go

```diff
@@ -56,12 +56,17 @@ func (c *Connection) beginCall(ctx context.Context, serviceName string) (*Outbou
 		return nil, err
 	}
 
+	headers := callHeaders{
+		CallerName: c.localPeerInfo.ProcessName,
```

Comment:
Oh interesting, so do we need to have the caller specify this explicitly? What is the value when a client makes this call (and the client does not have any services exposed)?